source/Halibut.Tests/BackwardsCompatibility/InvocationExceptionsAreMappedCorrectlyFixture.cs

```diff
@@ -12,13 +11,11 @@ namespace Halibut.Tests.BackwardsCompatibility
     public class InvocationExceptionsAreMappedCorrectlyFixture : BaseTest
     {
         [Test]
-        [TestCaseSource(typeof(ServiceConnectionTypesToTest))]
+        [LatestClientAndPreviousServiceVersionsTestCases(testNetworkConditions: false)]
         [FailedWebSocketTestsBecomeInconclusive]
-        public async Task OldInvocationExceptionMessages_AreMappedTo_ServiceInvocationHalibutClientException(ServiceConnectionType serviceConnectionType)
+        public async Task OldInvocationExceptionMessages_AreMappedTo_ServiceInvocationHalibutClientException(ClientAndServiceTestCase clientAndServiceTestCase)
         {
-            using (var clientAndService = await LatestClientAndPreviousServiceVersionBuilder
-                       .ForServiceConnectionType(serviceConnectionType)
-                       .WithServiceVersion(PreviousVersions.v5_0_429.ServiceVersion.ForServiceConnectionType(serviceConnectionType))
```

Comment:
This test used a specific version of Halibut that is later than the version used in `LatestClientAndPreviousServiceVersionsTestCases`. After [this discussion](https://octopusdeploy.slack.com/archives/C04JFJXECS1/p1690424471049419), we decided that v5_0_429 was not used anywhere after this PR, and we could delete that project.